src/schemas/Clinical/gqlTypeDefs.ts

```diff
@@ -184,7 +184,7 @@ export default gql`
   Collated Clinical Data Query Response
   """
   type ClinicalData {
-    programShortName: String!
+    programShortName: String
```

Comment:
Is it possible for us to return clinical data with a null programShortName? That seems like it shouldn't be possible...